config/clusterbuilder.yaml

```diff
@@ -1,16 +1,18 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: clusterbuilders.build.pivotal.io
+  name: clusterbuilders.kpack.io
 spec:
-  group: build.pivotal.io
+  group: kpack.io
   version: v1alpha1
   names:
     kind: ClusterBuilder
     singular: clusterbuilder
     plural: clusterbuilders
     shortNames:
-    - clstbldr
+    - ccb
+    - customclusterbuilders
```

Comment:
Would love to remove these as well, or have the only short names be `cb` and `cbs`.